writeengine/wrapper/writeengine.cpp

```diff
@@ -1690,34 +1929,70 @@ int WriteEngineWrapper::insertColumnRecs(const TxnID& txnid,
         int         curFbo = 0, curBio, lastFbo = -1;
 
         if (isFirstBatchPm && (totalRow == rowsLeft))
-        {}
+        {
```

Comment:
Could you ellaborate on this case? I don't see the meaning behind the argument sent to idbassert.
And what is the case we describe with the above if condition?

Reply:
This is hard to test case. I guess it can happen when running in cluster and node just went up and it is asked to insert values into an extent which does not exists on it right now.